components/ProgressView/src/MDCProgressView.h

```diff
@@ -86,4 +86,20 @@ IB_DESIGNABLE
          animated:(BOOL)animated
        completion:(void (^__nullable)(BOOL finished))completion;
 
+/**
+ Start the progress bar's indeterminate animation.
+ */
+- (void)startAnimating;
```

Comment:
Creating a `isAnimating` boolean property here would help making it testable through unit test.

Reply:
Looks like tests are still needed?